### ISSUE TITLE
UI usability - border contrast

### ISSUE THREAD

**Issue by bd0bd** [backdrop/backdrop-issues#1632]
I would like to propose to make a little bit more contrast colors or divide it by a line:
![u](https://cloud.githubusercontent.com/assets/13515302/13190718/c0226c08-d780-11e5-80df-a2a607032fbe.gif)

**Comment by BWPanda** [comment]
Is this still desired @bd0bd? If so, can you please expand on your request?

**Comment by oadaeh** [comment]
The background colors on the layout page are close enough together that a person with vision problems cannot tell the difference.

I have a friend who has set up several WordPress sites, but she is not a developer. She comes to Joseph and me when she gets into technical depths. We showed her Backdrop, and she has created a few sites using it, but she had problems in the beginning, because she could not see the differences in the background colors between the page, the regions, and the blocks. Joseph had to show her that they even existed, before she could effectively use that page, and she currently does so with difficulty.

I think in addition to making the colors easier to differentiate, adding borders (i.e., lines) to the various blocks would also help a lot.

**Comment by oadaeh** [comment]
I would add an accessibility label if we had one.

Also, this is related to #4941

**Comment by indigoxela** [comment]
I agree that Seven has some contrast problems. We could fix them one by one, but eventually it's time to consider a new admin theme.

In this case it's actually layout.admin.css. Should we fix it in Seven or in the layout module css?

**Comment by philsward** [comment]
> Should we fix it in Seven or in the layout module css?

If it's fixed in seven, do we run the risk of the same problem on other or future admin themes?

If the answer is yes, I would recommend layouts css.

**Comment by indigoxela** [comment]
Personally, I think we should keep the change as small and limited in scope as possible. Which means to only change the colors in layout.admin.css.

We'll probably run into the same "can we change core CSS without breaking existing stuff" discussion - like several times before. Keeping the impact to a minimum might help to get a solution more quickly. :wink:

**Comment by klonos** [comment]
Noting that we have similar contrast issues in the UI that we manage regions for flexible templates. Adjusted the issue title to denote that.
